recommendation/pom.xml

```diff
@@ -81,4 +84,31 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+      <profile>
+        <id>f8</id>
+        <properties>
+          <tag.version>v1</tag.version>
```

Comment:
We'll need to update the readme.md to include instructions on updating the tag here too. But it's ok to leave it this way for now.

Reply:
Ah crap. I had intended to make a note of that. let me update these things and push a new commit. Thanks @yanaga

Reply:
So, this change adds a large number of of additional lines to the pom.xml, therefore what are the "wins" for our end-user?  Where has this reduced complexity/learning curve proportionally?
In addition, the live & ready probes generated by F-m-p would break Istio 0.4.0, has that been corrected in later releases of Istio?  Istio 0.4.0 required the curl-based live/ready.

Reply:
@burrsutter added lines to the pom? so? What are we trying to prove with a small pom file? 

This is an *optional* maven profile for developers who wish to stay in their comfortable java land and not to know anything about docker files and kubernetes resources AKA our target enterprise audience.

liveness probes seemed to work fine for me on 0.5.x.

Reply:
Istio is an abstraction on top of deployments. If you have to do docker build to use Istio, you're being distracted with noise that has nothing to do with Istio. In any event, I'm not proposing changing your flow. I'm asking you to consider this completely *optional* usage that would expand the utility of this istio_tutorial to users who don't care about docker builds and for folks who deliver these types of tutorials that wish to demonstrate a much better developer experience.